mmcv/cnn/bricks/crf.py

```diff
@@ -0,0 +1,286 @@
+import torch
+import torch.nn as nn
+from typing import List, Optional
+
+
+class CRF(nn.Module):
```

Comment:
Suggest renaming `CRF` to `ConditionalRandomField`
```suggestion
class ConditionalRandomField(nn.Module):
```